frontends/ios/BitBoxApp/BitBoxApp.xcodeproj/project.pbxproj

```diff
@@ -464,7 +470,7 @@
 				INFOPLIST_KEY_UIApplicationSupportsIndirectInputEvents = YES;
 				INFOPLIST_KEY_UILaunchScreen_Generation = YES;
 				INFOPLIST_KEY_UISupportedInterfaceOrientations_iPad = "UIInterfaceOrientationPortrait UIInterfaceOrientationPortraitUpsideDown UIInterfaceOrientationLandscapeLeft UIInterfaceOrientationLandscapeRight";
-				INFOPLIST_KEY_UISupportedInterfaceOrientations_iPhone = "UIInterfaceOrientationPortrait";
```

Comment:
These changes have been made by Xcode automatically, not sure the reason or whether they should be reverted or not

Reply:
git blame will tell you the reason for this line, it's deliberate :) should definitely revert

Reply:
well git blame won't tell me why XCode changed it, I meant :D reverted!